frontend/src/pages/MetadataByType/MetadataByTypeBody.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  padding: 0.5em;
+`;
+
+const Tag = styled.div`
+  background-color: #eeeeee;
+  color: #595959;
+  font-size: 0.9em;
+  padding: 0.1em 0.7em;
+  display: inline-block;
+  border-radius: 1em;
+  margin: 0.3em;
+`;
+
```

Comment:
The css on my computer at "http://localhost:3000/viewData/dataType/Car%20history" doesn't seem to be fully done. The footer doesn't stay at the bottom of the page.